server/lib/search.js

```diff
@@ -76,7 +76,7 @@ export const TS_VECTOR = `
   to_tsvector('english', name)
   || to_tsvector('simple', slug)
   || to_tsvector('english', COALESCE(description, ''))
-  || COALESCE(array_to_tsvector(tags), '')
+  || COALESCE(to_tsvector(array_to_string(tags, ' ')), '')
```

Comment:
These two both seems to fail:

```sql
SELECT array_to_string(ARRAY[])
SELECT array_to_string(NULL)
```

Let's make this a bit more resiliant:

```suggestion
  || COALESCE(to_tsvector(array_to_string(COALESCE(tags::varchar[], ARRAY[]::varchar[]), ' ')), '')
```

Reply:
Done. Thanks for pointing it out. 👍🏽

Reply:
@Betree : Actually I realized that this one has a problem. When trying to execute this script we get, 

```
ERROR: functions in index expression must be marked IMMUTABLE
```
Will investigate this more. 🤔